contracts/deploy/039_wrapped_ousd.js

```diff
@@ -0,0 +1,71 @@
+const { deploymentWithProposal, withConfirmation } = require("../utils/deploy");
+
+// Deploy new staking implimentation contract with fix
```

Comment:
Typo here and I think this comment is from a previous deploy

Reply:
Removed, thanks!